### PR TITLE
Fix generating GDNative API struct for 1.1

### LINKED ISSUE #25425: gdnative_api_struct_.gen.cpp not populating next with api_1_1
**Godot version:**
3.1 alpha
de0d30655859a46eaf711e209fe20d9ca60a60f2

**OS/device including version:**
Windows 10 64 bit

**Issue description:**
gdnative_builder.py is not populating the .next member on api_struct. Below is the code generated by the builder, with my comment indicating where I believe the missing value is located:

extern const godot_gdnative_core_api_struct api_struct = {
	GDNATIVE_CORE,
	{1, 0},
	NULL,   //   should be (const godot_gdnative_api_struct *)&api_1_1
	4,
	gdnative_extensions_pointers,
	godot_color_new_rgba,
	godot_color_new_rgb,
	godot_color_get_r,
	godot_color_set_r,
	godot_color_get_g,
	godot_color_set_g,
	godot_color_get_b,

**Steps to reproduce:**
Build Godot with scons.

**Minimal reproduction project:**
None -- look at generated file

### PR DESCRIPTION
**Pull request by QbieShay**
fixes #25425

### CODE REVIEW

**Comment by starry-abyss** [comment]
The fix looks completely unrelated to the issue mentioned?

**Comment by QbieShay** [comment]
Yup sorry, I inverted two numbers

**Review by karroffel — APPROVED** [review]
I would like to make this more solid to handle future expansions better as well, but for 3.1 this should do the trick

**Comment by akien-mga** [comment]
Could you amend the commit log to be more explicit? It currently doesn't say what it actually does (in plain English), and references the wrong issue :)

Something like:
```
Fix generating GDNative API struct for 1.1

Fixes #25425.
```
would be great :)

**Comment by QbieShay** [comment]
Double woops, will fix this evening

**Comment by akien-mga** [comment]
Thanks!
